src/content/guides/output-management.md

```diff
@@ -184,33 +184,28 @@ As you might have noticed over the past guides and code example, our `/dist` fol
 
 In general it's good practice to clean the `/dist` folder before each build, so that only used files will be generated. Let's take care of that.
```

Comment:
```suggestion
In general it's good practice to clean the `/dist` folder before each build, so that only used files will be generated. Let's take care of that with [`output.clean`](/configuration/output/#outputclean) option.
```